assets/src/edit-story/stickers/beauty4Heart.js

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+function Beauty4Heart({ style }) {
+  return (
+    <svg
+      style={style}
+      width="32"
+      height="26"
```

Comment:
'width' and 'height' should be stripped out of the SVG so that just the viewBox is deciding width/height.

Reply:
👍  was going to suggest putting stickers in their own template-specific subdirs. 

It'd be nice to bake this into the sticker type/ID too for easier parsing/categorization, if possible e.g. `beauty/cta` instead of `beautyCta`.